docs/user-guide/gotchas.md

```diff
@@ -0,0 +1,9 @@
+# Gotchas
+
+## PIE
+
+In order to extract complex blocks, basic blocks, and instructions from position-idependent executables (PIE), OFRAK changes the virtual address of `CodeRegion`'s unpacked from the file format to match the virtual address used by the backend (Ghidra, Angr, etc.)
+
+This is because different backends load PIE files at different virtual addresses. For example Ghdira loads them at `0x10000` whereas Angr loads the mat `0x400000`. However, for file formats such as ELF, the virtual address taken from the header is `0x0`.
```

Comment:
```suggestion
This is because different backends load PIE files at different virtual addresses. For example Ghidra loads them at `0x10000` whereas Angr loads them at `0x400000`. However, for file formats such as ELF, the virtual address taken from the header is `0x0`.
```